bindings/nodejs/examples/offline_signing/example_prepared_transaction.json

```diff
@@ -2,109 +2,103 @@
     "essence": {
         "type": "Regular",
         "data": {
-            "inputs": [
-                {
-                    "type": "Utxo",
-                    "data": "9f8e51a1eca2c5b1c6b761e07b10d198ebd2dc1fb49f32048ebc9e544bbcc32d0100"
-                }
-            ],
-            "outputs": [
-                {
-                    "type": "SignatureLockedSingle",
-                    "data": {
-                        "address": {
-                            "type": "Ed25519",
-                            "data": "96f9de0989e77d0e150e850a5a600e83045fa57419eaf3b20225b763d4e23813"
-                        },
-                        "amount": 881000177
-                    }
-                },
-                {
-                    "type": "SignatureLockedSingle",
-                    "data": {
-                        "address": {
-                            "type": "Ed25519",
-                            "data": "f8208cdcda8b1afc710fbcb2e822fe70661c3213172189b845d49a64dd52a7a4"
-                        },
-                        "amount": 1000000
+            "network_id": 10859088168901003000,
+            "inputs": {
+                "inner": [
+                    {
+                        "type": "Utxo",
+                        "data": "0xbb2bdc323adf0e1ae8d8c35ae71d06bde7ff46978072a801c9279d901e5a68060200"
                     }
-                }
+                ],
+                "bounded": null
+            },
+            "inputs_commitment": [
+                18, 177, 93, 152, 247, 36, 206, 226, 89, 230, 20, 127, 45, 93,
+                60, 98, 96, 136, 48, 16, 181, 30, 150, 142, 133, 168, 210, 52,
+                218, 52, 121, 255
             ],
+            "outputs": {
+                "inner": [
+                    {
+                        "type": "Basic",
+                        "data": {
+                            "amount": 1000000,
```

Comment:
I believe this is outdated, should be a String in the latest version

Reply:
Correct! I'll update it.